packages/serve-cli/src/commands/proxy.ts

```diff
@@ -78,6 +78,7 @@ export const addCommand: AddCommand = (ctx, cli) =>
               },
             }
           : {}),
+        ...(polling ? { pollingInterval: polling } : {}),
```

Comment:
We can use the same name here ?

Reply:
I left it shorthand because the help explains it quite well:
```
--polling <intervalInMs>                schema polling interval in milliseconds (default: 10000, env: POLLING)
```

Reply:
We could use seconds orhuman-readable formats? 10s, 1min? 👀 
I really see no use-case for milliseconds

Reply:
I like that idea

Reply:
[Done.](https://github.com/ardatan/graphql-mesh/pull/7501/commits/8536867a7a8197708b3c527df7f5dab38d8b084f) Config file still uses milliseconds because it feels more natural to JS.